src/hub.js

```diff
@@ -273,16 +273,22 @@ const onReady = async () => {
           mediaStream.removeTrack(track);
         }
       }
-      NAF.connection.adapter.setLocalMediaStream(mediaStream);
+      if (NAF.connection.adapter && NAF.connection.adapter.setLocalMediaStream) {
+        NAF.connection.adapter.setLocalMediaStream(mediaStream);
+      }
       screenEntity.setAttribute("visible", sharingScreen);
     });
 
     document.body.addEventListener("blocked", ev => {
-      NAF.connection.entities.removeEntitiesOfClient(ev.detail.clientId);
+      if (NAF.connection.entities) {
```

Comment:
so it should be safe to remove this check?

Reply:
Yup, entities is also always defined.